demo/components/presentational/EstateForSale.test.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { shallow } from "enzyme";
+import EstateForSale from "./EstateForSale";
+
+describe("EstateForSale", () => {
+  it("renders the component", async () => {
+    const estate = {
+      id: -1,
```

Comment:
Maybe a random number would be better in case we ever add validation on the integers being positive?

Reply:
A random number will fail the snapshot test. I'll change for a positive one. Ok?

Reply:
Sorry, I meant an "arbitrary" number, not "random". Sounds good. Thanks!

Reply:
Done.

Reply:
Thanks! Resolving.